src/api/app/views/webui2/webui/monitor/_building_table.html.haml

```diff
@@ -0,0 +1,58 @@
+#building-list
+  - if worker_status.key?('building')
+    %table.responsive.table.table-sm.table-striped.table-bordered#building-table
```

Comment:
The `table` is not fully responsive
![Screenshot_2019-06-04 Open Build Service](https://user-images.githubusercontent.com/1212806/58887181-70d2b280-86e5-11e9-8f8f-745f33b1c62f.png)

Reply:
Please have another look.